### PR TITLE
Release native modules

### PR DESCRIPTION
**Pull request by wanglinsong**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -109,7 +109,7 @@ pipeline {
                     export GPG_TTY=${TTY}
 
                     cd presto
-                    unset MAVEN_CONFIG && ./mvnw -s ${WORKSPACE}/settings.xml -V -B -U -e -T2C deploy \
```
Comment (jaystarshot) [inline Jenkinsfile:112]: this atleast needs to have -T1C or it will be very slow

Comment (wanglinsong) [reply Jenkinsfile:112]: updated

**Comment by wanglinsong** [comment]
@tdcmeehan

File: Jenkinsfile (hunk 1 above)
Comment (wanglinsong) [reply Jenkinsfile:112]: updated

**Review by tdcmeehan — APPROVED** [review]

**Pull request merged**